static/ansi-to-html.ts

```diff
@@ -25,12 +25,14 @@
 
 // Converted from https://github.com/rburns/ansi-to-html
 // Includes patches from https://github.com/rburns/ansi-to-html/pull/84
+// Converted to typescript by MarkusJx
 
 'use strict';
```

Comment:
This can be removed, as 'use strict' is automatically emitted by the TypeScript compiler